src/Makefile.am

```diff
@@ -86,3 +86,19 @@ nodist_metainfo_DATA = \
 metainfo_in = $(patsubst %,%.in,$(nodist_metainfo_DATA))
 EXTRA_DIST += $(metainfo_in)
 CLEANFILES += $(nodist_metainfo_DATA)
+
+# -----------------------------------------------------------------------------
+#  Policykit
+#
+polkit_in_files  = src/cockpit/data/org.cockpit-project.cockpit-bridge.policy.in
+EXTRA_DIST += $(polkit_in_files)
+
+if WITH_POLKIT
+polkitdir        = $(datadir)/polkit-1/actions
+polkit_DATA = $(polkit_in_files:.policy.in=.policy)
```

Comment:
The difference is that previously the .polkit.in and Makefile.am files were in the same dir, but this isn't the case any more. I bet it's confused by the builddir vs. srcdir path..